src/app/shared/formly/formly.module.ts

```diff
@@ -1,209 +1,29 @@
-          component: CheckboxFieldComponent,
-          wrappers: ['form-field-checkbox-horizontal'],
-        },
-        { name: 'ish-captcha-field', component: CaptchaFieldComponent },
-        {
-          name: 'ish-fieldset-field',
-          component: FieldsetFieldComponent,
-        },
-        {
-          name: 'ish-radio-field',
-          component: RadioFieldComponent,
-          wrappers: ['form-field-checkbox-horizontal'],
-        },
-      ],
-      wrappers: [
-        { name: 'form-field-horizontal', component: HorizontalWrapperComponent },
-        { name: 'form-field-checkbox-horizontal', component: HorizontalCheckboxWrapperComponent },
-        { name: 'input-addon', component: InputAddonWrapperComponent },
-        { name: 'textarea-description', component: TextareaDescriptionWrapperComponent },
-        { name: 'tooltip', component: TooltipWrapperComponent },
-        { name: 'validation', component: ValidationWrapperComponent },
-        { name: 'description', component: DescriptionWrapperComponent },
-      ],
       extras: {
         lazyRender: true,
         showError: (field: FieldType) =>
           field.formControl?.invalid &&
           (field.formControl.dirty || field.options.parentForm?.submitted || !!field.field.validation?.show),
       },
-      extensions: [{ name: 'hide-if-empty-options-extension', extension: hideIfEmptyOptionsExtension }],
     }),
-    FormlySelectModule,
-    IconModule,
-    NgbPopoverModule,
-    ReactiveFormsModule,
-    TranslateModule,
-  ],
-  declarations: [
-    CaptchaFieldComponent,
-    CheckboxFieldComponent,
-    DescriptionWrapperComponent,
-    FieldsetFieldComponent,
-    FieldTooltipComponent,
-    HorizontalCheckboxWrapperComponent,
-    HorizontalWrapperComponent,
-    InputAddonWrapperComponent,
-    PlainTextFieldComponent,
-    RadioFieldComponent,
-    SelectFieldComponent,
-    TextareaDescriptionWrapperComponent,
-    TextareaFieldComponent,
-    TextInputFieldComponent,
-    TooltipWrapperComponent,
-    ValidationIconsComponent,
-    ValidationMessageComponent,
-    ValidationWrapperComponent,
-  ],
-  providers: [
-    {
-      provide: FORMLY_CONFIG,
-      multi: true,
-      useFactory: registerTranslateSelectOptionsExtension,
-      deps: [TranslateService],
-    },
-    {
-      provide: FORMLY_CONFIG,
-      multi: true,
-      useFactory: registerTranslatePlaceholderExtension,
-      deps: [TranslateService],
-    },
-    {
-      provide: FORMLY_CONFIG,
-      multi: true,
-      useFactory: registerPostWrappersExtension,
-      deps: [FormlyConfig],
-    },
-  ],
-  exports: [
-    CaptchaFieldComponent,
-    FormlyBaseModule,
-    SelectFieldComponent,
-    TextareaFieldComponent,
-    TextInputFieldComponent,
-    ValidationMessageComponent,
+    ExtensionsModule,
+    TypesModule,
+    WrappersModule,
   ],
+  exports: [FieldTooltipComponent, FormlyBaseModule, ValidationIconsComponent, ValidationMessageComponent],
```

Comment:
ComponentsModule instead of the 3 components - it will be easier for projects to add further components